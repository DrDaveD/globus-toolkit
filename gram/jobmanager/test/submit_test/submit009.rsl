&(executable=$(GLOBUS_LOCATION)/bin/grid-proxy-info)
 (arguments=-type)
