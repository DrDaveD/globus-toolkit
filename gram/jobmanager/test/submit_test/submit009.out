Proxy draft (pre-RFC) compliant limited proxy
